instrumentation/opentelemetry-instrumentation-tortoiseorm/src/opentelemetry/instrumentation/tortoiseorm/__init__.py

```diff
@@ -0,0 +1,316 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Instrument `tortoise-orm`_ to report SQL queries.
+Usage
+-----
+.. code:: python
+    from opentelemetry.instrumentation.tortoiseorm import TortoiseORMInstrumentor
+    from tortoise.contrib.fastapi import register_tortoise
+
+    TortoiseORMInstrumentor().instrument(tracer_provider=tracer)
+
+    register_tortoise(
+        app,
+        db_url=settings.db_url,
+        modules={"models": ["example_app.db_models"]},
+        generate_schemas=True,
+        add_exception_handlers=True,
+    )
+
+API
+---
+"""
+from typing import Collection
+
+import wrapt
+
+from opentelemetry import trace
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.tortoiseorm.package import _instruments
+from opentelemetry.instrumentation.tortoiseorm.version import __version__
+from opentelemetry.instrumentation.utils import unwrap
+from opentelemetry.semconv.trace import DbSystemValues, SpanAttributes
+from opentelemetry.trace import SpanKind
+from opentelemetry.trace.status import Status, StatusCode
+
+try:
+    import tortoise.backends.asyncpg.client
+
+    TORTOISE_POSTGRES_SUPPORT = True
+except ModuleNotFoundError:
+    TORTOISE_POSTGRES_SUPPORT = False
+
+try:
+    import tortoise.backends.mysql.client
+
+    TORTOISE_MYSQL_SUPPORT = True
+except ModuleNotFoundError:
+    TORTOISE_MYSQL_SUPPORT = False
+
+try:
+    import tortoise.backends.sqlite.client
+
+    TORTOISE_SQLITE_SUPPORT = True
+except ModuleNotFoundError:
+    TORTOISE_SQLITE_SUPPORT = False
+
+import tortoise.contrib.pydantic.base
+
+TORTOISE_PYDANTIC_MODEL_ATTRIBUTE = "pydantic.model"
```

Comment:
Avoid single use module level constants. Just use `"pydantic.model"` directly. Also remember to keep your public symbols to the minimum, only when strictly necessary.